tests/test_benchmark.py

```diff
@@ -49,7 +49,7 @@ def test_inference_no_configs(self):
         self.check_results_dict_not_empty(results.memory_inference_result)
 
     def test_inference_no_configs_only_pretrain(self):
-        MODEL_ID = "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english"
+        MODEL_ID = "sgugger/tiny-distilbert-classification"
```

Comment:
Would be nice to eventually move that to a model on `hf-internal-testing`; I'll upload the tokenizers shortly so that we may have tiny model from that org exclusively.

Reply:
Yes, that was quick fix to be able to merge this PR quickly.